client/app/src/Controller/Admin/FixtureController.php

```diff
@@ -193,7 +193,7 @@ public function retrieveFormData($form, $request): array
     {
         $submitted = $form->getData();
         $courtDate = $request->get('court-date') ? new \DateTime($request->get('court-date')) : new \DateTime();
-        $deputyEmail = $request->query->get('deputy-email', sprintf('original-%s-deputy-%s@fixture.com', strtolower($submitted['deputyType']), mt_rand(1000, 9999)));
+        $deputyEmail = $request->query->get('deputy-email', sprintf('original-%s-deputy-%s@fixture.com', null !== $submitted['deputyType'] ? strtolower($submitted['deputyType']) : null, mt_rand(1000, 9999)));
```

Comment:
It would be more PHP to use `is_null()` rather than `null === x`.

Reply:
Ah thanks. Will change that :)

Reply:
Sorry @jamesrwarren I think you missed this one.